### ISSUE TITLE
 Fix 'test_feeds/canonical/test_missing_tags' tests for canonical and debian in 4.2

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1552]
### Issue information

| Related issue |
|--|
| Closes https://github.com/wazuh/wazuh-qa/issues/1531|

When it comes to starting with the test fix, there is some information that can help you:
```
Module: Vulnerability Detector
Support Team:  "binary beasts"
Target: Manager   
OS: Linux
```
I add some information obtained while researched the logs that failed.

## Case 1: Canonical

|  Type | Description  |
|--|--|
|Test Path | test_feeds/canonical/test_missing_tags_canonical_feed
| Consistent | yes
| Test Execution | 3/3 executions failed
| Cases Fails | 30
| Summary | On `test_missing_tags_canonical_feed` when it called `check_feed_imported_successfully` method is called. The problem seems to be a Parser error when reading the file.
| Error message |  "Could not find the message: 'Ubuntu Bionic' feed finished successfully" 

## Case 2: Debian 

|  Type | Description  |
|--|--|
|Test Path | test_feeds/debian/test_missing_tags_debian_feed
| Consistent | no
| Test Execution | 2/3 executions failed
| Cases Fails | 39
| Summary | On `remove_tag_feed` when it called `read_xml_file` method is called. The problem seems to be a Parser error when reading the file.
| Error message |  "Could not find the message: 'Debian Buster' feed finished successfully" 

In order to finish this issue the following tasks should be fulfilled:
- [ ] Research of fails.
- [ ] Apply Fix 
- [ ] Full Green/ Full Yellow  in `test_feeds/canonical/test_missing_tags_canonical_feed` for 3 times.
- [ ] Full Green/ Full Yellow  in `test_feeds/debian/test_missing_tags_debian_feed` for 3 times.
- [ ] Documentation of any important change done for these tests or the used tools.

**Comment by mdengra** [comment]
### 2021-07-15

Test results with the modules `sca`, `syscollector` and `rootcheck` modules disabled in the `ossec.conf`: 

### Case 1
| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_missing_tags_canonical_feed_local_dm_r1.log](https://github.com/wazuh/wazuh-qa/files/6824118/test_missing_tags_canonical_feed_local_dm_r1.log)| 2021-07-15 | Miguel | :yellow_circle: 
|[test_missing_tags_canonical_feed_local_dm_r2.log](https://github.com/wazuh/wazuh-qa/files/6824119/test_missing_tags_canonical_feed_local_dm_r2.log)| 2021-07-15 | Miguel | :yellow_circle: 
|[test_missing_tags_canonical_feed_local_dm_r3.log](https://github.com/wazuh/wazuh-qa/files/6824120/test_missing_tags_canonical_feed_local_dm_r3.log)| 2021-07-15 | Miguel | :yellow_circle: 

### Case 2
| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_missing_tags_debian_feed_local_dm_r1.log](https://github.com/wazuh/wazuh-qa/files/6824127/test_missing_tags_debian_feed_local_dm_r1.log)| 2021-07-15 | Miguel | :green_circle: 
|[test_missing_tags_debian_feed_local_dm_r2.log](https://github.com/wazuh/wazuh-qa/files/6824131/test_missing_tags_debian_feed_local_dm_r2.log)| 2021-07-15 | Miguel | :yellow_circle: 
|[test_missing_tags_debian_feed_local_dm_r3.log](https://github.com/wazuh/wazuh-qa/files/6824132/test_missing_tags_debian_feed_local_dm_r3.log)| 2021-07-15 | Miguel | :yellow_circle:

**Comment by damarisg** [comment]
This issue is closed by #1605

**Comment by damarisg** [comment]
This issue is closed by #1605 

## Test Results from `1531-full-yellow-vuln-det`

| Test Executions | Tests | Date  | By  | Status | 
|--|--|--|--|--|
|[YellowResultsMissingTagsCanonicalFeed.log](https://github.com/wazuh/wazuh-qa/files/6856826/YellowResultsMissingTagsCanonicalFeed.log)| test_feeds/canonical/test_missing_tags_canonical_feed |2021-07-21| Seyla| 🟡 
|[YellowResultsMissingTagsCanonicalFeed2.log](https://github.com/wazuh/wazuh-qa/files/6856829/YellowResultsMissingTagsCanonicalFeed2.log)|  test_feeds/canonical/test_missing_tags_canonical_feed |2021-07-21| Seyla| 🟡 
|[YellowResultsMissingTagsCanonicalFeed3.log](https://github.com/wazuh/wazuh-qa/files/6856830/YellowResultsMissingTagsCanonicalFeed3.log)| test_feeds/canonical/test_missing_tags_canonical_feed| 2021-07-21| Seyla| 🟡 
|[YellowResultsMissingTagsDebianFeed.log](https://github.com/wazuh/wazuh-qa/files/6856837/YellowResultsMissingTagsDebianFeed.log)| test_feeds/debian/test_missing_tags_debian_feed |2021-07-21| Seyla| 🟡 
|[YellowResultsMissingTagsDebianFeed2.log](https://github.com/wazuh/wazuh-qa/files/6856856/YellowResultsMissingTagsDebianFeed2.log)| test_feeds/debian/test_missing_tags_debian_feed |2021-07-21| Seyla| 🟡 
|[YellowResultsMissingTagsDebianFeed3.log](https://github.com/wazuh/wazuh-qa/files/6856954/YellowResultsMissingTagsDebianFeed3.log) | test_feeds/debian/test_missing_tags_debian_feed |2021-07-21| Seyla| 🟡 


| Reference | Status | 
|--|--|
| 🟡 | Pass with Warnings |
